config/packages/sql.yaml

```
@@ -0,0 +1,36 @@
+sql:
```

YAMLlint (1.37.1)

[error] wrong new line character: expected \n

(new-lines)